.github/workflows/sanity-check.yml

```diff
@@ -51,4 +51,4 @@ jobs:
         continue-on-error: false
         with:
           script: |
-            throw new Error('Commit has no DBZ-xxx prefiix')
+            throw new Error('Commit has no DBZ-XXXX message prefix')
```

Comment:
While I agree this is a bit better wording, please let's not mix it in here.

Reply:
Fixed.